publication-model-testing/src/main/java/no/unit/nva/model/testing/PublicationContextBuilder.java

```diff
@@ -239,7 +241,7 @@ private static String randomSeriesNumber() {
 
     private static Anthology randomChapter() {
         return new Anthology.Builder()
-            .withId(randomUri())
+            .withId(randomPublicationId())
```

Comment:
This fixes a hidden bug in existing tests, where mocked data contained invalid publication context IDs that were never validated (until now).